gatsby-node.ts

```diff
@@ -0,0 +1,46 @@
+import path from 'path'
+
+import { copyLibFiles } from '@builder.io/partytown/utils'
+import type { GatsbyNode } from 'gatsby'
+
+import { apiSchema } from './src/server'
+
+export const onPreInit: GatsbyNode['onPreInit'] = async ({ reporter }) => {
+  reporter.info('Copying Partytown Files')
+
+  await copyLibFiles(path.resolve('./public/~partytown'))
+}
+
```

Comment:
I think this bit got removed in #433.

cc @igorbrasileiro 

```suggestion
import type { GatsbyNode } from 'gatsby'

import { apiSchema } from './src/server'

```

Reply:
nice catch!